docs/body_file.md

```diff
@@ -8,138 +8,156 @@ tags:
   -  Text
   -  Articles that need to be expanded
 ---
-Body file is a timeline format introduced by the
-[SleuthKit](sleuthkit.md).
 
-Version 2 (used by SleuhtKit 2):
+# Body File
 
-    MD5 | path/name | device | inode | mode_as_value | mode_as_string | num_of_links | UID | GID | rdev | size | atime | mtime | ctime | block_size | num_of_blocks
+_Body file may also be referred to as "bodyfile", however official documentation refers to it as body file (two separate words)._
 
-Version 3 (used by SleuhtKit 3 and later):
+The **body file** format is a delimiter-separated output timeline format introduced by the [SleuthKit](sleuthkit.md). Body files are pipe (`|`) delimited and are referred to as an "intermediate file", as they are not sorted chronologically and are often staged for post-processing. Subsequent timeline sorting is done via the [mactime](mactime.md) tool.
 
-    MD5|name|inode|mode_as_string|UID|GID|size|atime|mtime|ctime|crtime
+Body files are often generated by the following tools:
+<!-- Need to insert links to these tools -->
+* fls
+* ils
 
-The times are reported in a POSIX (or UNIX) timestamp.
+## Data & Fields
 
-### Shortcomings
+All times within a body file are reported in UNIX time format. Lines that start with `#` are ignored and treated as comments.
```

Comment:
`Lines that start with `#` are ignored and treated as comments.` do you have a reference to this?

Reply:
I see that is mentioned here https://wiki.sleuthkit.org/index.php?title=Body_file